### DISCUSSION TITLE
Crashlytics SDK Updagrade

### DISCUSSION THREAD

**Discussion by mym0404** [invertase/react-native-firebase#4562] [Q&A] [answered]
The Firebase team released the new standalone Crashlytics SDK.

I saw this [news](https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=android&authuser=0) that requests about the bumping Crashlytics version and installation method change.

In this library, the Crashlytics is set correctly already? I can see the legacy Gradle plugin or other things about Crashlytics like that following yet.

```groovy
maven { url 'https://maven.fabric.io/public' }
```

**Comment by andersonaddo** [comment, marked as answer]
We don't use fabric anymore.
https://github.com/invertase/react-native-firebase/pull/3580
That maven snippet is probably from an older version of our docs/test suite.

**Reply by mym0404** [reply]
Thank you ~.~
